cfgov/unprocessed/js/routes/on-demand/simple-chart/chart-hooks.js

```diff
@@ -83,7 +83,7 @@ const hooks = {
   },
 
   enforcement_reliefBarTooltipFormatter() {
-    return `<b>${ this.x }</b><br/>Total relief: <b>$${ this.y.toLocaleDateString() }</b>`;
+    return `<b>${ this.x }</b><br/>Total relief: <b>$${ this.y.toLocaleString() }</b>`;
```

Comment:
`this.y` is a number, not a date. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString